apps/fz_http/lib/fz_http_web/views/device_view.ex

```diff
@@ -1,13 +1,13 @@
 defmodule FzHttpWeb.DeviceView do
   use FzHttpWeb, :view
 
-  alias FzHttp.Configurations, as: Conf
+  import Wrapped.Cache
 
   def can_manage_devices?(user) do
-    has_role?(user, :admin) || Conf.get!(:allow_unprivileged_device_management)
+    has_role?(user, :admin) || cache().get!(:allow_unprivileged_device_management)
   end
 
   def can_configure_devices?(user) do
-    has_role?(user, :admin) || Conf.get!(:allow_unprivileged_device_configuration)
+    has_role?(user, :admin) || cache().get!(:allow_unprivileged_device_configuration)
```

Comment:
I feel like we put authorization too deep into view logic making it hard to understand if somebody wants to dig in and see how we handle it.

Reply:
Sounds good. The sooner we get client apps, the sooner only `:admins` will be using the web portal. Imagine how much mess we can remove then 🤩

Reply:
Maybe we can make client apps on top of webview? There is no need to maintain UI for every app separately, if we can just make it web-based